static/src/javascripts/projects/common/modules/commercial/dfp/private/render-advert.js

```diff
@@ -149,6 +151,9 @@ define([
 
             function callSizeCallback() {
                 var size = slotRenderEvent.size.join(',');
+                if (size === '0,0') {
```

Comment:
I wanted to keep `sizeCallbacks[adSize.fluid]` (which will become `sizeCallbacks.fluid`), but I agree this is ugly. Ideally I would like to find the correct element in `adSize` based on `slotRenderEvent.size`. Perhaps the whole `ad-sizes` module needs rethinking.